apps/aecore/config/config.exs

```diff
@@ -53,7 +53,7 @@ config :aecore, :aewallet,
   path: Path.absname(aewallet_path)
 
 config :aecore, :persistence,
-  path: Path.absname(persistence_path),
+  path: persistence_path |> Path.absname() |> Path.join("//"),
```

Comment:
According to the Elixir documentation "Note that any trailing slash is removed when joining."

Reply:
This is done so that the path is created correctly. With 2 slashes we get the desired result.